### DISCUSSION TITLE
Add a feature that allows you to report bugs in-game for stable.

### DISCUSSION THREAD

**Discussion by mixernix** [ppy/osu#14294] [Ideas] [answered]
A proper guide alongside it or even a bug report template would be helpful. Not everyone has github, so a lot of things will be missed or not even reported. Make bug reporting more accessible.

**Comment by bdach** [comment, marked as answer]
This is not the right repository for stable issues. And stable is feature-locked.

If you're not willing to use github then I believe direct email is the way to go.

**Reply by mixernix** [reply]
Fair point.
